disco/extensions/upgrade_simulation/upgrades/automated_thermal_upgrades.py

```diff
@@ -42,7 +43,8 @@ def determine_thermal_upgrades(
     logger.info("Initial simulation parameters: %s", initial_simulation_params)
     create_plots = thermal_config["create_plots"]
     # start upgrades
-    simulation_params = reload_dss_circuit(dss_file_list=[master_path], commands_list=None, **initial_simulation_params)
+    initial_dss_file_list = [os.path.abspath(master_path)]

```

Comment:
I made these absolute paths, cause the redirects weren't working others. Please let me know if better to have relative or absolute paths.

Reply:
Will the absolute paths show up in the `upgraded_master.dss` file? If so, this isn't great. It will make the files non-transportable.

Reply:
Yes, absolute paths show up.

Okay, makes sense, so should all the paths be relative to the output directory?